src/validations/createTargetValidations.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  areaLength: {
+    presence: true,
+  },
+  title: {
+    presence: true,
+  },
+};
```

Comment:
isn't topic also presence: true?

Reply:
Yes! It is controlled outside these validations